### DISCUSSION TITLE
CSS/JS Bundles separated per-template

### DISCUSSION THREAD

**Discussion by molotow11** [uicrooks/shopify-theme-lab#91] [Q&A] [answered]
Hello, can we have CSS/JS Bundles separated per-template? Like Shopify Packer has?

**Comment by sergejcodes** [comment, marked as answer]
No, that's not possible out of the box. For it to work you would have to rewrite a lot of logic. Plus it would make the developer experience a lot more complex.

**Reply by molotow11** [reply]
Thank you, got it.
